praw/reddit.py

```diff
@@ -509,7 +509,7 @@ def patch(self, path, data=None):
 
         """
         data = self.request("PATCH", path, data=data)
-        return self._objector.objectify(data)
+        # return self._objector.objectify(data)
```

Comment:
Should this return data?

Reply:
My mistake.

I guess I should write some tests for these now that they may not get used elsewhere in the codebase.